internal/pkg/skuba/kubernetes/clientset.go

```diff
@@ -28,7 +27,7 @@ import (
 func GetAdminClientSet() (*clientset.Clientset, error) {
 	client, err := kubeconfigutil.ClientSetFromFile(skuba.KubeConfigAdminFile())
 	if err != nil {
-		return nil, errors.Wrap(err, "could not load admin kubeconfig file")
```

Comment:
Why remove the wrap? Do you have the error messages before/after this specific line?

Reply:
Because it's already wrapped here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/kubeconfig/kubeconfig.go#L76 I don't think we need to wrap it again.

Reply:
+1, thanks.